CMakeLists.txt

```
@@ -251,7 +256,8 @@ include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 find_package(gflags COMPONENTS shared)
 find_package(glog REQUIRED)
 
-find_library(FMT fmt)
+set(fmt_SOURCE AUTO)
```

Commit once rebased on https://github.com/facebookincubator/velox/pull/3149
```suggestion
set_source(fmt)
```